<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerSelection</class>
 <widget class="QDialog" name="PlayerSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="player_selection_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>611</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>101</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="num_player_2">
      <property name="text">
       <string>2 Players</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="num_player_3">
      <property name="text">
       <string>3 Players</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="num_player_4">
      <property name="text">
       <string>4 Players</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="num_player_5">
      <property name="text">
       <string>5 Players</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="num_player_6">
      <property name="text">
       <string>6 Players</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="num_player_7">
      <property name="text">
       <string>7 Players</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="num_player_8">
      <property name="text">
       <string>8 Players</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>311</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLineEdit" name="player_name_2">
      <property name="text">
       <string>Player 2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="player_name_3">
      <property name="text">
       <string>Player 3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="player_name_4">
      <property name="text">
       <string>Player 4</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="player_name_5">
      <property name="text">
       <string>Player 5</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="player_name_6">
      <property name="text">
       <string>Player 6</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="player_name_7">
      <property name="text">
       <string>Player 7</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="player_name_8">
      <property name="text">
       <string>Player 8</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="play_button">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>490</y>
     <width>151</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>PLAY!</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="player_name_1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Player 1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>131</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Player 1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
